python/sglang/srt/layers/attention/base_attn_backend.py

```diff
@@ -39,6 +39,7 @@ def init_forward_metadata_capture_cuda_graph(
     def init_forward_metadata_replay_cuda_graph(
         self,
         bs: int,
+        num_kv_heads: int,
```

Comment:
Please revert this interface change. This value is a constant so we do not need to pass it for every function call. We can get it in `__init__`  function of the attention backends.